README.md

```diff
@@ -13,22 +14,27 @@ Repository to describe, develop, document and test the Blockchain Public Address
 * Describe, develop, document and test the APIs (with 1-2 Telcos)  
 * Started: June 2023
-* Location: virtually  
 
-## Meetings
-* Meetings are held virtually
-* Schedule: Thursday, 13:00-14:00 CET/CEST (12:00 UTC, 11:00 UTC during European DST)
-* Meeting link: [Microsoft Teams Meeting](https://teams.microsoft.com/l/meetup-join/19%3ameeting_ZmM4N2QwNTEtMmQ5ZS00ZDU4LWI1MDMtZDQ0OWEyYWJmMDRk%40thread.v2/0?context=%7b%22Tid%22%3a%229744600e-3e04-492e-baa1-25ec245c6f10%22%2c%22Oid%22%3a%2219764050-b5d5-4991-9f15-d10905a94c08%22%7d)
+## Release Information
+<!-- Use/uncomment one or multiple the following options -->
+<!-- Pre-releases of this sub project are available in https://github.com/camaraproject/§repo_name§/releases -->
+<!-- The latest public release is available here: https://github.com/camaraproject/§repo_name§/releases/latest -->
+<!-- For changes see [CHANGELOG.md](https://github.com/camaraproject/§repo_name§/blob/main/CHANGELOG.md) -->
+  * [Latest Release](https://github.com/camaraproject/BlockchainPublicAddress/releases/latest)
+  * Release Information: Link to wiki
+    * Note: Please be aware that the project will have frequent updates to the main branch. There are no compatibility guarantees associated with code in any branch, including main, until a new release is created. For example, changes may be reverted before a release is created. **For best results, use the latest available release**.
+    * **The first available release and version of Blockchain Public Address API is 0.1.0.** There are expected changes in upcoming releases. It is addressed for early implementations, but not a stable version yet.
+    * Release 0.1.0 of the API is available within the [release-0.1.0 branch](https://github.com/camaraproject/BlockchainPublicAddress/tree/release-0.1.0):
+      - API definition **with inline documentation**:
+      - OpenAPI [YAML spec file](https://github.com/camaraproject/BlockchainPublicAddress/blob/release-0.1.0/code/API_definitions/blockchain_public_address.yaml)
+      - [View it on ReDoc](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/camaraproject/BlockchainPublicAddress/release-0.1.0/code/API_definitions/blockchain_public_address.yaml&nocors)
+      - [View it on Swagger Editor](https://editor.swagger.io/?url=https://raw.githubusercontent.com/camaraproject/BlockchainPublicAddress/release-0.1.0/code/API_definitions/blockchain_public_address.yaml)
```

Comment:
Proposal to align the Release Information with other APIs (I hope that the indentation will work as expected):

```suggestion
* Note: Please be aware that the project will have frequent updates to the main branch. There are no compatibility guarantees associated with code in any branch, including main, until a new release is created. For example, changes may be reverted before a release is created. **For best results, use the latest available release**.
* **The first initial release and version of Blockchain Public Address API is 0.1.0 and is available within the [release-0.1.0 branch](https://github.com/camaraproject/BlockchainPublicAddress/tree/release-0.1.0) ** There are expected changes in upcoming releases. It is addressed for early implementations, but not a stable version yet.
* API definition of Blockchain Public Address API is 0.1.0 (with inline documentation):
  - OpenAPI [YAML spec file](https://github.com/camaraproject/BlockchainPublicAddress/blob/release-0.1.0/code/API_definitions/blockchain_public_address.yaml)
  - [View it on ReDoc](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/camaraproject/BlockchainPublicAddress/release-0.1.0/code/API_definitions/blockchain_public_address.yaml&nocors)
  - [View it on Swagger Editor](https://editor.swagger.io/?url=https://raw.githubusercontent.com/camaraproject/BlockchainPublicAddress/release-0.1.0/code/API_definitions/blockchain_public_address.yaml)
* All releases and pre-releases are available here https://github.com/camaraproject/BlockchainPublicAddress/releases
* For changes see [CHANGELOG.md](https://github.com/camaraproject/BlockchainPublicAddress/blob/main/CHANGELOG.md)
```